bin/repair/startup.py

```diff
@@ -0,0 +1,11 @@
+import os
+from os.path import dirname
+from nose.importer import Importer
+bin_dir = os.path.split(__file__)[0]
+component_dir = os.path.join(bin_dir, "..", "..", "..")
```

Comment:
Scripts in repair can't use bin/startup because of this?

Reply:
They can't do `from .. import startup` because `bin` isn't a package, and I couldn't make `bin` into a package without the nose import trick.

I did getthis working by stacking the two import hacks, like this:

```
import os
import sys
bin_dir = os.path.split(__file__)[0]
package_dir = os.path.join(bin_dir, "..")
sys.path.append(os.path.abspath(package_dir))
import startup
from core.scripts import CheckContributorNamesInDB
CheckContributorNamesInDB().run()
```

I decided it was better to keep the startup code in another file, even though it's nearly a copy of the file in the parent directory.